p2panda-rs/src/schema/validate/blob_piece.rs

```diff
@@ -0,0 +1,64 @@
+// SPDX-License-Identifier: AGPL-&3.0-or-later
+
+use crate::operation::plain::{PlainFields, PlainValue};
+use crate::schema::validate::error::BlobPieceError;
+
+const MAX_BLOB_PIECE_LENGTH: usize = 256;
+
+/// Checks "data" field of operations with "blob_piece_v1" schema id.
+///
+/// 1. It must be less than `MAX_BLOB_PIECE_LENGTH`
+pub fn validate_data(value: &String) -> bool {
+    value.len() <= MAX_BLOB_PIECE_LENGTH
```

Comment:
Nice, good reminder that `String::len` actually returns the bytes of the string, not the number of characters 😆